### ISSUE TITLE
Decide on Tech Stack

### ISSUE THREAD

**Issue by SomeMoosery** [COVID-19-electronic-health-system/Corona-tracker#4]
Once we have an idea decided upon, we need to decide upon a tech stack (these could technically be done in parallel as well). So far, the consensus seems to be around:

**Frontend:** React

**Backend:** serverless via Blockstack, Django, Express/Node

**Database:** Blockstack Radisk (need to read into this myself but seems like Mongo), Mongo, PostGIS (Postgres extension - I'm not sure many people know this and neither do I, but if we want to optimze for storing primarily geography-based information, this is great at that).

Personally, my vote kind of forks based on another decision. If we want to develop a decentralized app, I would vote **React, Blockstack**. If we feel like decentralized isn't necessary (or we want to put it as more of a stretch goal which would require reshaping architecture down the line), I'd vote **React, Express/Node (or Go, if anyone else wants but I think that's just me lol), MongoDB**.

Let's discuss and try to come up with a decision ASAP so we're all on the same page and can start working towards a cohesive goal.

**Comment by SomeMoosery** [comment]
Since time is of the essence here, I think we should settle on building a React app on the Blockstack platform (using things like Gaia, Radiks, etc). I haven't worked with Blockstack before, but it seems like a great solution for going serverless (aside from AWS/Azure which people may grow skeptical of in an increasingly-paranoid time) and seems to plug in extremely easily with React.

I'm personally going to go through [this](https://docs.blockstack.org/develop/zero_to_dapp_1.html) to get a better sense for how to build on the platform.
